tests/packages/mlx/conftest.py

```diff
@@ -0,0 +1 @@
+from pkgfixtures import host_with_saved_yum_state
```

Comment:
Because it's a workaround, we repeat this comment when we import such fixtures.

```suggestion
# Explicitly import package-scoped fixtures (see explanation in pkgfixtures.py)
from pkgfixtures import host_with_saved_yum_state
```